internal/service/privatelinkendpointserviceserverless/data_source_privatelink_endpoints_service_serverless.go

```diff
@@ -76,18 +76,12 @@ func PluralDataSource() *schema.Resource {
 	}
 }
 
-func dataSourceMongoDBAtlasPrivateLinkEndpointsServiceServerlessRead(ctx context.Context, d *schema.ResourceData, meta any) diag.Diagnostics {
-	// Get client connection.
-	conn := meta.(*config.MongoDBClient).Atlas
+func dataSourcePluralRead(ctx context.Context, d *schema.ResourceData, meta any) diag.Diagnostics {
+	connV2 := meta.(*config.MongoDBClient).AtlasV2
 	projectID := d.Get("project_id").(string)
 	instanceName := d.Get("instance_name").(string)
 
-	options := &matlas.ListOptions{
-		PageNum:      d.Get("page_num").(int),
```

Comment:
should we deprecate page_num and iterms_per_page ?

Reply:
I would leave this for future project to remove pagination in different resources and avoid breaking changes for now.

Reply:
i mean to flag the properties as deprecated, I think we should do it here as we're doing with other data sources.
Not saying to sunset/delete them here, that will be later and the real breaking change.

Reply:
Understood! I set them as Deprecated in the latest commit

Reply:
great.  Can you better use or create a new constant in deprecation.go file in constant package, so it's easier to search for deprecated params, like constant.DeprecationParamByDateWithReplacement ...  , etc. ?

Reply:
Done!